test1.js

```diff
@@ -3,3 +3,10 @@
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+
+var foo77 = '' ;
+
+if(foo77  === "baaaaaaaaaar") {
```

Comment:
Codacy has a fix for the issue: [Replace `(foo77·` with `·(foo77`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5796)
```suggestion
if (foo77 === "baaaaaaaaaar") {
```